src/components/CalendarNav.vue

```diff
@@ -164,6 +166,43 @@ export default {
     getYearGroupIndex(year) {
       return Math.floor(year / _yearGroupCount);
     },
+    getMonthItems(yearIndex) {
+      const { month: thisMonth, year: thisYear } = this.pageForDate(new Date());
+      return this.locale.getMonthDates().map((d, i) => {
+        const month = i + 1;
+        return {
+          label: this.locale.format(d, this.masks.navMonths),
+          ariaLabel: this.locale.format(d, 'MMMM YYYY'),
+          isActive: month === this.month && yearIndex === this.year,
+          isCurrent: month === thisMonth && yearIndex === thisYear,
+          isDisabled: !this.validator({ month, year: yearIndex }),
+          click: () => this.monthClick(month),
+        };
+      });
+    },
+    getYearItems(yearGroupIndex) {
+      const { _, year: thisYear } = this.pageForDate(new Date());
+      const startYear = yearGroupIndex * _yearGroupCount;
+      const endYear = startYear + _yearGroupCount;
+      const items = [];
+      for (let year = startYear; year < endYear; year += 1) {
+        let enabled = false;
+        for (let month = 1; month < 12; month++) {
```

Comment:
@nathanreyes 
Maybe a mistake here ?
It should be "month <= 12" because actually last month of year is missing.
If min-date is "2020-12-03" I can't chose 2020 in years popover the 12th month is missing